### PR TITLE
Home page implemented

### PR DESCRIPTION
**Pull request by govardhanreddyc79**
Home Page implemented

### CODE REVIEW

**Review by manojncsoni007 — COMMENTED** [review]
Looks good

File: index.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+
+.category {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    width: 15rem;
+    height: 15rem;
+    border-radius: 5px;
+    background-color: lightgray;
```
Comment (som749898) [inline index.css]: use variable for different color scheme

File: index.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+}
+.category-cards {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: center;
+    align-items: center;
+    gap:5rem;
+}
+.cat-image {
+    width: 168px;
```
Comment (som749898) [inline index.css:30]: use rem instead of px

File: index.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    font-size: 2rem;
+    height: 117px;
```
Comment (ranitasahaa1230) [inline index.css:32]: Use rems over here

**Review by som749898 — COMMENTED** [review]
Have added a few suggestions
Rest looks good

File: index.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    width: 45rem;
+    height: 40rem;
+}
+.new-arrivals {
+    width: 25rem;
+    height: 20rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: lightgray;
```
Comment (ranitasahaa1230) [inline index.css:48]: You can put all the colors in the root variables

File: index.css [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    gap:2rem;
+    padding: 1rem;
+}
+.deal-image {
+    width: 180px;
+    font-size: 2rem;
+    height: 140px;
```
Comment (ranitasahaa1230) [inline index.css:55]: Use rems for responsiveness too

**Review by ranitasahaa1230 — APPROVED** [review]
Suggested a few changes, Rest looks good

File: index.css (hunk 1 above)
Comment (govardhanreddyc79) [reply index.css]: sure, Thanks

File: index.css (hunk 2 above)
Comment (govardhanreddyc79) [reply index.css:30]: okay

File: index.css (hunk 3 above)
Comment (govardhanreddyc79) [reply index.css:32]: okay

File: index.css (hunk 4 above)
Comment (govardhanreddyc79) [reply index.css:48]: sure

File: index.css (hunk 5 above)
Comment (govardhanreddyc79) [reply index.css:55]: okay

**Pull request merged**